pkgs/os-specific/linux/xmm7360-pci/default.nix

```diff
@@ -11,6 +11,20 @@ stdenv.mkDerivation rec {
     sha256 = "1wdb0phqg9rj9g9ycqdya0m7lx24kzjlh25yw0ifp898ddxrrr0c";
   };
 
+  patches = [
```

Comment:
Note I did ship the patches here in-tree, as upstream seems to be somewhat unresponsive on merging these in.